src/main/java/net/bootsfaces/component/dataTable/DataTableCore.java

```diff
@@ -86,6 +86,22 @@ public boolean isBorder() {
 	public void setBorder(boolean _border) {
 		getStateHelper().put(PropertyKeys.border, _border);
 	}
+	/**
+	 * If set, this will add a <caption> tag in table declaration 
+	 */
+	public void setcaption(String _caption) {
+	        getStateHelper().put(PropertyKeys.caption, _caption);
+	}
+	
+	
+	/**
+	 * If set, this will surround the table by a border. Defaults to true. <P>
+	 * @return Returns the value of the attribute, or "-1", if it hasn't been set by the JSF file.
+	 */
+	public String getCaption() {
+	        return (String) getStateHelper().eval(PropertyKeys.caption, "-1");
```

Comment:
Why not use `null` here as the default?

Reply:
I tried to copy the behaviour of other attributes but it seems I took a size attribute as a reference. You are right I should have used the attributes null  using "-1" doesn't make much sense :disappointed:.